src/Bigcommerce/Api/Client.php

```diff
@@ -1140,4 +1142,102 @@ public static function getRequestsRemaining()
 
         return intval($limit);
     }
+
+    /**
+     * Get a single shipment by given id.
+     *
+     * @param $orderID
+     * @param $shipmentID
+     * @return mixed
+     */
+    public static function getShipment($orderID, $shipmentID)
+    {
+        return self::getResource('/orders/' . $orderID . '/shipments/' . $shipmentID, 'Shipment');
+    }
+
+    /**
+     * Get shipments for a given order
+     *
+     * @param $orderID
+     * @param array $filter
+     * @return mixed
+     */
+    public static function getShipments($orderID, $filter = array())
+    {
+        $filter = Filter::create($filter);
+        return self::getCollection('/orders/' . $orderID . '/shipments' . $filter->toQuery(), 'Shipment');
+    }
+
+    /**
+     * Create shipment
+     *
+     * @param $orderID
+     * @param $object
+     * @return hash|bool|mixed
+     */
+    public static function createShipment($orderID, $object)
+    {
+        return self::createResource('/orders/' . $orderID . '/shipments', $object);
+    }
+
+    /**
+     * Update shipment
+     *
+     * @param $orderID
+     * @param $shipmentID
+     * @param $object
+     * @return hash|bool|mixed
+     */
+    public static function updateShipment($orderID, $shipmentID, $object)
+    {
+        return self::updateResource('/orders/' . $orderID . '/shipments/' . $shipmentID, $object);
+    }
+
+    /**
+     * Delete the given shipment.
+     *
+     * @param $orderID
+     * @param $shipmentID
+     * @return hash|bool|mixed
+     */
+    public static function deleteShipment($orderID, $shipmentID)
+    {
+        return self::deleteResource('/orders/' . $orderID . '/shipments/' . $shipmentID);
+    }
+
+    /**
+     * Delete all Shipments for the given order.
+     *
+     * @param $orderID
+     * @return hash|bool|mixed
+     */
+    public static function deleteAllShipmentsForOrder($orderID)
```

Comment:
The only name I'm not sold on is this one--but it's not bad, and I can't think of anything more clear.